src/utils/notify/index.js

```diff
@@ -0,0 +1,35 @@
+const admin = require('firebase-admin');
+const { query: Hasura } = require('../hasura');
+const { getFcmToken } = require('./queries');
+
+const notify = (entityTypeId, entityId, actorId, notifierIds) => {
+  const fcmToken = Hasura(getFcmToken, {
+    id: { _eq: notifierIds },
+  }).then((res) => res.json().data);
```

Comment:
Hasura is an async function, please look into what it is returning & how it has been used in other places. res.json() will throw error.